tests/test_asset.py

```diff
@@ -1,10 +1,13 @@
+import json
 import pathlib
+from typing import Dict, List, Optional
 from unittest import mock
 
 import pystac
 import pytest
+import requests_mock as req_mock
 
-from up42 import asset, host
+from up42 import asset, auth, host
```

Comment:
nitpick: although not necessary, but consider importing `auth as up42_auth` to be consistent with imports in other modules.